docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -318,7 +318,7 @@ To do so you can use the `-t` flag:
    root@host:~# su - bareos -s /bin/sh -c "bareos-sd -t"
    root@host:~# bareos-fd -t
    There are configuration warnings:
```

Comment:
The CollectStatistics warning will be printed by `bareos-dir -t`; we should reorder them such that that check happens last.  Otherwise we need to move the warning to the appropriate command.